docs/jax-101/03-vectorization.ipynb

```diff
@@ -0,0 +1,376 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zMIrmiaZxiJC"
+   },
+   "source": [
+    "# Automatic vectorization in JAX\n",
+    "\n",
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax/blob/master/docs/jax-101/03-vectorization.ipynb)\n",
+    "\n",
+    "*Authors: TODO*\n",
+    "\n",
+    "In the previous section we discussed JIT compilation via the `jax.jit` function. This notebook discusses another of JAX's transforms: vectorization via `jax.vmap`."
```

Comment:
Suggestion: provide a link to the "previous section" for better UX (as in "[previous section](....awesome link....)")

Reply:
Same issue as in notebook 2... we don't know if people are reading this as a notebook on github or as a page in the docs, so it's not clear which link to include.

Reply:
Gotcha @jakevdp thank you